src/content/docs/logs/logs-context/configure-logs-context-nodejs.mdx

```diff
@@ -20,9 +20,182 @@ redirects:
   - /docs/logs/enable-log-management-new-relic/configure-logs-context/configure-logs-context-nodejs
 ---
 
-Logs in context for the Node.js agent connects your logs and APM data in New Relic. Bringing all of this data together in a single tool helps you quickly get to the root cause of an issue and find the log lines that you need to identify and resolve a problem.
+APM logs in context connects your logs with all of your telemetry data for your apps, hosts, and other entities. Bringing all of this data together in a single tool helps you quickly:
 
-## Set up your Node.js app [#configure-node-js]
+* Cut through the noise of thousands of logs when troubleshooting time-critical issues, so you automatically see only the most relevant logs.
+* Navigate within multiple types of telemetry data, and have the data correlate back to the original issue.
+* Easily drill down into more detailed information from the same place in the UI.
+* Find the log lines that you need to identify and resolve a problem.
+
+For more information, including examples, learn how to get started with [APM logs in context](/docs/apm/new-relic-apm/getting-started/get-started-logs-context).
+
+## Automatic logs in context options [#automatic]
+
+By enabling application logging, you can tell your APM agent to automatically collect and contextualize your logs. Application logging is not enabled by default at this time, but that may change in the future.
```

Comment:
application_logging.enabled is defaulted to true.  however forwarding and local log decorating are off by default at the moment